### PR TITLE
#feat - button badge and card component added.

### PR DESCRIPTION
**Pull request by surajmore98**
- documentation added for component:
     Badge
     Button
     Card

- link added to side navigation for previous component pages.

### CODE REVIEW

File: src/documentation/component/badge.html [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+                <li class="side-bar-content">
+                    <a>Typography</a>
+                </li>
+                <li class="side-bar-content">
+                    <a>Margin</a>
+                </li>
+                <li class="side-bar-content">
+                    <a>Padding</a>
+                </li>
+            </ul> -->
```
Comment (romabulani) [inline src/documentation/component/badge.html]: You can remove the comments if not required

File: src/documentation/component/button.html [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                <li class="side-bar-content">
+                    <a>Typography</a>
+                </li>
+                <li class="side-bar-content">
+                    <a>Margin</a>
+                </li>
+                <li class="side-bar-content">
+                    <a>Padding</a>
+                </li>
+            </ul> -->
```
Comment (romabulani) [inline src/documentation/component/button.html]: Here also you can remove the comments

File: src/documentation/component/card.html [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+                <li class="side-bar-content">
+                    <a>Typography</a>
+                </li>
+                <li class="side-bar-content">
+                    <a>Margin</a>
+                </li>
+                <li class="side-bar-content">
+                    <a>Padding</a>
+                </li>
+            </ul> -->
```
Comment (romabulani) [inline src/documentation/component/card.html]: You can remove the comments from this file also

**Review by romabulani — APPROVED** [review]
Rest all looks good to merge

**Review by ravikumar1002 — APPROVED** [review]
Looks good to merge, suraj

**Pull request merged**